code/lib/blocks/src/components/ArgsTable/types.ts

```diff
@@ -47,3 +47,5 @@ export interface Args {
 }
 
 export type Globals = { [name: string]: any };
+
+export type HidableColumn = 'descrption' | 'default' | 'control';
```

Comment:
@sir-captainmorgan21 as @MichaelArestad in the comment above, don't forget to update this one as well.

Reply:
I suck. Thanks!